spec/models/retweet_spec.rb

```diff
@@ -1,5 +1,16 @@
 require 'rails_helper'
 
 RSpec.describe Retweet, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+
+# ASSOCIATIONS SPECS
+  context "associations" do
+    it { should belong_to (:tweet) }
+    it { should belong_to (:user) }
+  end
+
+  #---------------------------------------------------------------------------------------------------------
+# VALIDATION SPEC
+  context "validations" do
+    #it {should validate_uniqueness_of(:user).scoped_to(:tweet)}
+  end
```

Comment:
Missed uniqueness validitation test, you could have test it without shoulda matcher